lib/service/autoconfig/ispdb.php

```diff
@@ -36,7 +36,18 @@ public function __construct(Logger $logger) {
 
 	private function queryUrl($url) {
 		try {
-			$xml = @simplexml_load_file($url);
+			$content = @file_get_contents($url, false, stream_context_create([
+				'http' => [
+					'timeout' => 7
+				]
+			]));
+			if ($content !== False) {
```

Comment:
just another nitpick, but I think we ususaly use ``false`` instead of ``False`` :see_no_evil: